generate-java.sh

```diff
@@ -2,6 +2,7 @@
 
 dir_resolve()
 {
+"$1"!="" && mkdir -p "$1"
```

Comment:
Huh, I misread the go generator. Thanks for catching my error – and for making them similar.